packages.config

```diff
@@ -3,4 +3,6 @@
   <package id="NUnit" version="2.6.4" targetFramework="net40" />
   <package id="NUnit.Runners" version="2.6.4" />
   <package id="FsCheck" version="2.0.3" />
+  <package id="NUnit" version="3.0.0-beta-3" targetFramework="net45" />
```

Comment:
the old nunit is used in unit tests projects.
These proj are going to be converted to xunit instead of upgrade to nunit 3 in coreclr branch ( see #699 ) temporary or not.
Now nunit 3 final is release, so maybe is easier to upgrade the suite to nunit 3 final in all projects (usefull for coreclr too), but that's another pr